tasks.py

```diff
@@ -792,6 +792,12 @@ def test(ctx):
     run("KUBEBUILDER_ASSETS={} go test -short -race ./...".format(kubebuilder_assets), echo=True)
 
 
+@task
+def linttasks(ctx):
+    """Check if 'tasks.py' has being correctly formatted with 'black' linter."""
```

Comment:
Can we have this command perform the actual lint instead of just checking? It's going to be easier to know what to fix.

Reply:
since the fix is just about calling "black", maybe what we need is CI failing on "missing linting task.py with `black tasks.py`"

Reply:
right, which is what the `inv checkchanges` execution is supposed to do:

Just add 
`inv checkchanges --action="run inv linttasks"`
after the execution.

If checkchanges is not doing what is supposed to do, we'll file an issue to take care of it, but I'd keep the same format for ci: apply the changes, check that nothing changed.

Reply:
Ahh cool, then I will do that thanks!